src/main/ipc-handlers.ts

```diff
@@ -108,12 +122,28 @@ export const initializeIpcHandlers = (): void => {
 						}
 					}
 				}
-			} else if (desktop_platform === 'linux') {
-				// TODO: Platform checking on Linux
-				// Set 'game_platform.available' true and setup data if platform is available, do nothing otherwise
-				continue;
 			}
+			return DefaultWindowsGamePlatforms;
+		} else if (desktop_platform === 'linux') {
+			for (const key in DefaultLinuxGamePlatforms) {
+				const game_platform = DefaultLinuxGamePlatforms[key];
+				if (game_platform.key === GamePlatform.STEAM) {
+					// TODO: Check and support other types of steam installations; Flatpak
+					try {
+						const buff = readFileSync(homedir() + '/.steam/registry.vdf');
+						const steamVdf = parse(buff.toString()) as {Registry:{HKCU:{Software:{Valve:{Steam:{Apps:object}}}}}};
+						//tries to find Among Us's Steam Id in the .vdf-file
+						if ("945360" in steamVdf["Registry"]["HKCU"]["Software"]["Valve"]["Steam"]["Apps"]) {
+							game_platform.available = true;
+						}
+					} catch(e) {
+						/* empty */
+					}
+				}
+			}
+			return DefaultLinuxGamePlatforms;
+		} else {
+			return DefaultWindowsGamePlatforms;
```

Comment:
Better to return null and handle with null check where message invoked?

Reply:
In terms of best-practice, probably. In the new version I have in progress this is basically fixed so don't worry about it (I build up a list of available platforms/installs so I return an empty list if none are found).